### DISCUSSION TITLE
Rainbow miner

### DISCUSSION THREAD

**Discussion by hubert4567** [RainbowMiner/RainbowMiner#2364] [Q&A]
Hi, im quite new . I have question about rainbow miner :  Is it profitable ? Can i eanr like 20$ a day with not the best computer power ? I mean this script does more profitable mining yes  ?

**Comment by hubert4567** [comment]
can i run it on server ? where there is no good gpu ?

**Reply by RainbowMiner** [reply]
If you run it on a server, only CPU mining will be possible. Depending on the CPU, you could expect a return around 0.05-0.3$
